cmd/mimir/help.txt.tmpl

```diff
@@ -549,6 +549,10 @@ Usage of ./cmd/mimir/mimir:
     	HTTP server listen address.
   -server.http-listen-port int
     	HTTP server listen port. (default 8080)
+  -server.tls-cipher-suites string
+    	Comma-separated list of cipher suites to use. If blank, the default Go cipher suites will be used.
+  -server.tls-min-version string
+    	Minimum TLS version to use. If blank, the Go TLS minimum version will be used.
```

Comment:
```suggestion
    	Minimum TLS version to use. If blank, the Go TLS minimum version is used.
```

Reply:
This is a string. What are accepted values for this string? Can we document them?

Reply:
"VersionTLS10", "VersionTLS11", "VersionTLS12", "VersionTLS13"
https://github.com/weaveworks/common/pull/259